src/server/game/Movement/PathGenerator.cpp

```diff
@@ -904,7 +904,7 @@ float PathGenerator::Dist3DSqr(G3D::Vector3 const& p1, G3D::Vector3 const& p2) c
     return (p1 - p2).squaredLength();
 }
 
-void PathGenerator::ReducePathLenghtByDist(float dist)
+void PathGenerator::ReducePathLenghtByDist(const float dist, Unit* target)
```

Comment:
this method is called for now only at 2 points and for each of those cases, if i'm not mistaken, unit will never be null.